snuba/web/db_query.py

```diff
@@ -388,6 +388,39 @@ def get_query_cache_key(formatted_query: FormattedQuery) -> str:
     return md5(force_bytes(formatted_query.get_sql())).hexdigest()
 
 
+def check_sorted_sql_key_in_cache(
+    formatted_query_sorted: Optional[FormattedQuery], reader: Reader
+) -> bool:
+    cache_partition = _get_cache_partition(reader)
+    if formatted_query_sorted:
+        key_sorted = get_query_cache_key(formatted_query_sorted)
+        percentage = state.get_config(
+            "sort_sql_fields_and_conditions_rollout_percentage", 0
+        )
+        assert isinstance(percentage, int)
+        if percentage > hash_to_probability(key_sorted):
```

Comment:
Should this be `>` or `<`? If we have an experiment set to "10%", by convention that means 10% is _in_ the experiment, and 90% is the control.

Reply:
Rewritten to `hash_to_probability(key_sorted) < percentage` for clarity. This means that if the query's random probability is *within* the experiment percentage range (e.g. 10%), we check the sorted cache.